assets/src/styles/blocks/ColumnsEditor.scss

```diff
@@ -1,52 +1,52 @@
 .columns-image-container {
   position: relative;
+}
 
-  .buttons-overlay {
-    position: absolute;
-    z-index: 2;
-    align-items: center;
-    justify-content: center;
-    width: 100%;
-    height: 100%;
-    display: flex;
+.buttons-overlay {
```

Comment:
I'll move this to a separate file later. Didn't change anything inside of `.buttons-overlay`, it all still worked and works for the other block too.

That's why you should avoid needlessly putting classes inside of other classes, it makes the code way less reusable.